doc/whats_new.rst

```diff
@@ -107,6 +107,8 @@ API
 
     - A new option ``set_env`` has been added to :func:`mne.set_config` that defaults to ``False`` in 0.13 but will change to ``True`` in 0.14, by `Eric Larson`_
 
+    -  Added :class:`mne.decoding.Vectorizer` to convert MNE transformer output into two dimensions as required in scikit-learn pipeline. By `Asish Panda`_
```

Comment:
to convert the output of an MNE transformer into two dimensions so as to pipeline it to scikit-learn estimators.